python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -384,13 +384,9 @@ def test_dataframe_index_rename(axis):
 
     # `pandas` can support indexes with mixed values. We throw a
     # `NotImplementedError`.
-    expect = pdf.rename(mapper={1: "x", 2: "y"}, axis=axis)
-
     with pytest.raises(NotImplementedError):
         got = gdf.rename(mapper={1: "x", 2: "y"}, axis=axis)
 
-    assert_eq(expect, got)
```

Comment:
Does this cause issues? I thought the `with pytest.raises` short circuits the rest of the test but I could be mistaken.

Reply:
Looks like it doesn't short circuit as I previously, thought. Apologies for leading you down the wrong path!

Reply:
It does not. The other assertions will raise correctly if failed and the test passes normally all the other asserts are True and this raises the correct error.